Makefile

```diff
@@ -21,7 +15,7 @@ endif
 
 setup: venv_check
 	python -m pip install cookiecutter
-	-rm -rf baking-test
+	rm -rf baking-test
```

Comment:
That `-` makes this line optional. I think if the baking test directory isn't already there it will stop at this line and not generate the project.

Reply:
not sure I'm following. I think the intention here is to both reuse existing Makefile recipes. 

The `-f` flag should prevent this recipe from failing in case the `baking-test` doesn't exist (first time after checkout). This is already working in the GH action FWIW. 

Should not pose a problem, but LMK please if I'm overlooking something.

Reply:
It looks like it works without the optional flag.